build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.2'
+        classpath 'com.android.tools.build:gradle:7.0.1'
```

Comment:
I am asked to updated to `7.0.2`

Reply:
Okay weird sentence....
I updated to 7.0.2 (although I don't find any changelogs for that version at https://developer.android.com/studio/releases/gradle-plugin#7-0-0).

Should be fine now 😄

Reply:
asked by Android Studio :sweat_smile: